app/styles/partials/footer.scss

```diff
@@ -2,6 +2,9 @@
   display: flex;
   min-height: 100vh;
   flex-direction: column;
+  .main-container {
```

Comment:
@sumedh123 this isn't the right way to do this. This would affect all the other pages. (Pages which do not have a side-rail and hence don't have an overflow issue). 

Also, we want this fix only for smaller screen sizes.

So, 
- This styles should be applied only to pages that have a left rail
- The styles should be applied depending on the screenwidth. If the screen is wide enough, don't disturb semantic's default width/sizing.